pkg/kubelet/network/plugins.go

```diff
@@ -217,6 +218,13 @@ func (plugin *NoopNetworkPlugin) Init(host Host, hairpinMode kubeletconfig.Hairp
 	if err := utilsysctl.New().SetSysctl(sysctlBridgeCallIPTables, 1); err != nil {
 		glog.Warningf("can't set sysctl %s: %v", sysctlBridgeCallIPTables, err)
 	}
+	if val, err := utilsysctl.New().GetSysctl(sysctlBridgeCallIP6Tables); err == nil {
```

Comment:
Should we log the error returned from GetSysctl if it cannot find the module?

Reply:
Thanks for the review, this PR was changed to not show the error if it cannot find the module.
As dcbw pointed out in an earlier review, 
"Does this sysctl even exist if the kernel is booted with IPv6 disabled or compiled out? Not sure if we care about that case, but people still do this when they don't want IPv6, and I believe those make the sysctl simply not present and thus will always error spuriously."

So in the case where it can't find it, it just continues.